.github/workflows/r-test.yml

```diff
@@ -0,0 +1,74 @@
+name: testthat
+
+on:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    
+    env:
+      R_REMOTES_NO_ERRORS_FROM_WARNINGS: true
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up R
+        uses: r-lib/actions/setup-r@v2
+        
+      - name: Install Ubuntu dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y libcurl4-openssl-dev libharfbuzz-dev libfribidi-dev
+
+      - name: Install R dependencies
+        run: |
+          install.packages('remotes')
+          install.packages('devtools')
+          remotes::install_deps(dependencies = TRUE)
+          remotes::install_cran("rcmdcheck")
+          remotes::install_cran("httr")
+          remotes::install_github("Sage-Bionetworks/dccvalidator")
+          devtools::install()
+        shell: Rscript {0}
+
+      - name: Run tests
+        run: |
+          install.packages('testthat')
+          testthat::test_dir('tests/testthat/')
```

Comment:
Could we look into whether this command already runs the tests? https://github.com/Sage-Bionetworks/stopadforms/blob/12dd87b518f0140141470f64b8faa612f9eda599/.github/workflows/R-CMD-check.yaml#L93

Reply:
I don't know the answer, but based on [this](https://cran.r-project.org/web/packages/rcmdcheck/readme/README.html) it seems like it could...

Reply:
So, i'll double check, but i'm fairly sure not (or if it did, it wasnt failing on them) because the CI was passing for that, despite failing tests